### ISSUE TITLE
ACL Word template does not pass ACL Pubcheck

### ISSUE THREAD

**Issue by tnaumann** [acl-org/aclpubcheck#59]
There is a formatting issue where the [ACL Word template](https://2023.aclweb.org/calls/style_and_formatting/) does not pass ACL Pubcheck.

Specifically, use of the ACL Word template does not appear to be parsable by the PDF parser. Due to this marker is not populated (left as None) in https://github.com/acl-org/aclpubcheck/blob/main/aclpubcheck/formatchecker.py#L328-L350.

**Comment by crux82** [comment]
Hi Tristan

Sorry for the stupid question but: have you tried converting the Word file to PDF and uploading it?

Bests

Danilo

**Comment by tnaumann** [comment]
Yes, for clarity this is after the template has been exported / printed as a PDF (i.e. am not trying to use the PDF parsing utility on .doc / .docx). 

E.g., see [ClinicalNLP/papers/72.pdf](https://github.com/ACL-2023/Clinical-NLP/blob/22ed704b68c06165136a02ee53d575e2c19a22dc/papers/72.pdf) from this year's proceedings.
